config.json

```diff
@@ -0,0 +1,6 @@
+
+{
```

Comment:
Please change the default to the following. I want to commit the "prod" defaults to the repo:

```javascript
{
 "PLATFORM": "platform.cloud.coveo.com",
 "PUSH": "push.cloud.coveo.com",
 "REDIRECT_URI": "https://zapier.com/dashboard/auth/oauth/return/APP-ID/"
}
```

Reply:
I meant use `https://zapier.com/dashboard/auth/oauth/return/APP-ID/` literally. 
We don't want to commit our private app id in the repo.